assets/templates/geography-start.tmpl

```diff
@@ -0,0 +1,43 @@
+{{ $lang := .Language }}
```

Comment:
Not sure why you've assigned the language prop into a local variable here. This is usually useful to avoid having long prop names or if you are using a range and want to easily access another prop. I'll leave this one to you if you want to refactor at a later date

Reply:
I was actually just following another template I had looked at as an example.
Refactored out now.